.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
It seems through the rest of the project you are using 2 spaces for indentation, whereas here you are using 4. It should be uniform throughout, 2 seems to be the most common convention though this may vary depending on your workplace. Whichever you use it should be consistent throughout